src/PrestaShopBundle/Resources/config/services/core/form/command_builder.yml

```diff
@@ -7,21 +7,30 @@ services:
     arguments:
       - !tagged core.product_command_builder
 
+  prestashop.core.form.command_builder.product.update_product_commands_builder:
+    class: PrestaShop\PrestaShop\Core\Form\IdentifiableObject\CommandBuilder\Product\UpdateProductCommandsBuilder
+    tags:
+      - { name: 'core.product_command_builder', priority: 10 }
```

Comment:
Just a question about this priority, it means it is executed before the other builders for relations right?

Reply:
yes, I think it makes sense to execute this command first, no?
Im abit worried about the status though, but maybe its ok :thinking:

Reply:
Yeah that's what worried me as well, but it should be fine since the "activation rules" are based on the product field mainly not its relations